{"name": "test_forgotpassword_invalid_otp1_error", "status": "passed", "attachments": [{"name": "stdout", "source": "2e69f4f0-e3ec-45e4-8500-a19455ef9b25-attachment.txt", "type": "text/plain"}], "start": 1757605896521, "stop": 1757605897950, "uuid": "67687e50-5960-4962-8600-042021630f79", "historyId": "6ec9ad9310dcc6217263c941384037c4", "testCaseId": "6ec9ad9310dcc6217263c941384037c4", "fullName": "tests.test_forgotpassword#test_forgotpassword_invalid_otp1_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_forgotpassword"}, {"name": "host", "value": "OTSI-Lap80"}, {"name": "thread", "value": "87688-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_forgotpassword"}]}